### ISSUE TITLE
Off-machine node fails with NO OUTPUT FILE PRODUCED when transcoding -> Linux Server

### ISSUE THREAD

**Issue by janky5ive** [HaveAGitGat/Tdarr#1132]
Bug Description:
Node transcoding fails and produces NO OUTPUT FILE PRODUCED error. The node is able to fully complete transcode when performing audio-only CPU tasks. Confirmed node supports QSV (plugin for video transcoding requires). Transcode process works on on-machine node. Appears very similar to Bug#362  https://github.com/HaveAGitGat/Tdarr/issues/362. Similar to issue mentioned in Bug #362 from Scaredycrow. 

To Reproduce

Set up docker (linux) server, webui, and node on the machine hosting the media library
Share the transcode cache and media library directories
Map these shares to drives on the off-machine (windows) node
Add these paths to the node config
Change the node and server ip in the node config to their respective values
Spin up a transcode worker on the off-machine node

Expected behavior
The off-machine node should translate the paths correctly so that it can read and write to the transcode cache and media library.

Screenshot:
![2024-12-14 20_50_25-Tdarr](https://github.com/user-attachments/assets/f6bb7084-43e2-4402-9001-ba006f992a40)

Config Files:
[Tdarr_Node_Config.json](https://github.com/user-attachments/files/18138806/Tdarr_Node_Config.json)
[Tdarr_Server_Config.json](https://github.com/user-attachments/files/18138807/Tdarr_Server_Config.json)

Job Report:
[Job Report.txt](https://github.com/user-attachments/files/18138809/Job.Report.txt)

Log Files:
[Off-Machine Node Log.txt](https://github.com/user-attachments/files/18138810/Off-Machine.Node.Log.txt)
[Server-log.txt](https://github.com/user-attachments/files/18138811/Server-log.txt)

 - OS:
        >Server: Terramaster 5.1.131
        >Node: Windows 11 Home version 23H2
 - Browser [Chrome]
 - Version
        >on-machine - server: 2.27.02
        >off-machine - node: 2.27.02

Additional Information:
During my debugging process, I changed the path translation for the (transcode cache) temp folder from a rootfolder+directories -> to addressing the temp folder directly. For consistency, I tried to re-capture all logs and config after I made the change to addressing the temp folder directly. No change in behavior occured before/after making the path translation change.

**Comment by dennix85** [comment]
Error initializing an MFX session

Your gpu is not set up properly for your docker container

So yes it will not produce an output

**Comment by janky5ive** [comment]
> Error initializing an MFX session
> 
> Your gpu is not set up properly for your docker container
> 
> So yes it will not produce an output

Maybe I didnt communicate well enough.... The docker container running on the server allows for GPU accelerated transcoding on the server node. The problem I have is on the windows node, which is NOT running in a docker container.

If it is indeed a problem with the windows node GPU needing to be setup for the linux server docker container, is there any guidance you can provide?

**Comment by dennix85** [comment]
Nah should run fine on windows is there a Intel gpu in your windows node?
Because that is what your error saying.
It cannot initialize the gpu.

**Comment by janky5ive** [comment]
Yes, I should have included the windows node specs. There is an Intel Xe iGPU and a dedicated Nvidia GPU.

**Comment by dennix85** [comment]
İs the igpu enabled? And available in your device manager?

**Comment by janky5ive** [comment]
At work now. I will check tonight. Appreciate the help

**Comment by dennix85** [comment]
If it is not available then it's probably disabled in the bios,
If it is enabled then force the tdarr executables and the ffmpeg and handbrake executables
That are included with tdarr to use the Intel gpu via your desktop settings > graphics

**Comment by janky5ive** [comment]
Checked BIOS, no option to enable iGPU. Booted and in windows Settings > Display > Graphics... "Custom Options for apps" i selected the Tdarr node exe file and override system GPU decision to force iGPU. Still does not work. Attached is the job report and Node Log. Node log still shows a "No output file produced" error. Curiously, when running the flow on a file that only needs a 2ch audio track made (no video transcode), the windows node can run every step just fine.
[Node Log after Win Node fail.txt](https://github.com/user-attachments/files/18218776/Node.Log.after.Win.Node.fail.txt)
[Job Report after Win Node Fail.txt](https://github.com/user-attachments/files/18218777/Job.Report.after.Win.Node.Fail.txt)

**Comment by dennix85** [comment]
Did you enforce the ffmpeg and handbrake executables also to the igpu?
Same way you did for the node executable? 
They should be in tdarr folder as well.

**Comment by janky5ive** [comment]
Forced the ffmpeg and handbrakeCLI to the iGPU. works now :D 

Thank you, oh wise one! Merry Christmas!
